File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/MultiResultViewLayPaInfo.java
package nta.med.data.model.ihis.cpls;

public class MultiResultViewLayPaInfo {
	private String suname;
    private String suname2;
    private String sex;
    private Integer yearAge;
    private Integer monthAge;
    private String gubun;
    private String admDictNm;
    private String birth;
    private String tel;
    private String tel1;
    private String telHp;
    private String email;
    private String zipCode1;
    private String zipCode2;
    private String address1;
    private String address2;
    private String inpJaewonCheck;
	public MultiResultViewLayPaInfo(String suname, String suname2, String sex,
			Integer yearAge, Integer monthAge, String gubun, String admDictNm,
			String birth, String tel, String tel1, String telHp, String email,
			String zipCode1, String zipCode2, String address1, String address2,
			String inpJaewonCheck) {
		super();
		this.suname = suname;
		this.suname2 = suname2;
		this.sex = sex;
		this.yearAge = yearAge;
		this.monthAge = monthAge;
		this.gubun = gubun;
		this.admDictNm = admDictNm;
		this.birth = birth;
		this.tel = tel;
		this.tel1 = tel1;
		this.telHp = telHp;
		this.email = email;
		this.zipCode1 = zipCode1;
		this.zipCode2 = zipCode2;
		this.address1 = address1;
		this.address2 = address2;
		this.inpJaewonCheck = inpJaewonCheck;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public Integer getYearAge() {
		return yearAge;
	}
	public void setYearAge(Integer yearAge) {
		this.yearAge = yearAge;
	}
	public Integer getMonthAge() {
		return monthAge;
	}
	public void setMonthAge(Integer monthAge) {
		this.monthAge = monthAge;
	}
	public String getGubun() {
		return gubun;
	}
	public void setGubun(String gubun) {
		this.gubun = gubun;
	}
	public String getAdmDictNm() {
		return admDictNm;
	}
	public void setAdmDictNm(String admDictNm) {
		this.admDictNm = admDictNm;
	}
	public String getBirth() {
		return birth;
	}
	public void setBirth(String birth) {
		this.birth = birth;
	}
	public String getTel() {
		return tel;
	}
	public void setTel(String tel) {
		this.tel = tel;
	}
	public String getTel1() {
		return tel1;
	}
	public void setTel1(String tel1) {
		this.tel1 = tel1;
	}
	public String getTelHp() {
		return telHp;
	}
	public void setTelHp(String telHp) {
		this.telHp = telHp;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getZipCode1() {
		return zipCode1;
	}
	public void setZipCode1(String zipCode1) {
		this.zipCode1 = zipCode1;
	}
	public String getZipCode2() {
		return zipCode2;
	}
	public void setZipCode2(String zipCode2) {
		this.zipCode2 = zipCode2;
	}
	public String getAddress1() {
		return address1;
	}
	public void setAddress1(String address1) {
		this.address1 = address1;
	}
	public String getAddress2() {
		return address2;
	}
	public void setAddress2(String address2) {
		this.address2 = address2;
	}
	public String getInpJaewonCheck() {
		return inpJaewonCheck;
	}
	public void setInpJaewonCheck(String inpJaewonCheck) {
		this.inpJaewonCheck = inpJaewonCheck;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/MultiResultViewLayPaInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class MultiResultViewLayPaInfo
	{
		private String _suname;
		private String _suname2;
		private String _sex;
		private String _yearAge;
		private String _monthAge;
		private String _gubun;
		private String _admDictNm;
		private String _birth;
		private String _tel;
		private String _tel1;
		private String _telHp;
		private String _email;
		private String _zipCode1;
		private String _zipCode2;
		private String _address1;
		private String _address2;
		private String _inpJaewonCheck;

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String Suname2
		{
			get { return this._suname2; }
			set { this._suname2 = value; }
		}

		public String Sex
		{
			get { return this._sex; }
			set { this._sex = value; }
		}

		public String YearAge
		{
			get { return this._yearAge; }
			set { this._yearAge = value; }
		}

		public String MonthAge
		{
			get { return this._monthAge; }
			set { this._monthAge = value; }
		}

		public String Gubun
		{
			get { return this._gubun; }
			set { this._gubun = value; }
		}

		public String AdmDictNm
		{
			get { return this._admDictNm; }
			set { this._admDictNm = value; }
		}

		public String Birth
		{
			get { return this._birth; }
			set { this._birth = value; }
		}

		public String Tel
		{
			get { return this._tel; }
			set { this._tel = value; }
		}

		public String Tel1
		{
			get { return this._tel1; }
			set { this._tel1 = value; }
		}

		public String TelHp
		{
			get { return this._telHp; }
			set { this._telHp = value; }
		}

		public String Email
		{
			get { return this._email; }
			set { this._email = value; }
		}

		public String ZipCode1
		{
			get { return this._zipCode1; }
			set { this._zipCode1 = value; }
		}

		public String ZipCode2
		{
			get { return this._zipCode2; }
			set { this._zipCode2 = value; }
		}

		public String Address1
		{
			get { return this._address1; }
			set { this._address1 = value; }
		}

		public String Address2
		{
			get { return this._address2; }
			set { this._address2 = value; }
		}

		public String InpJaewonCheck
		{
			get { return this._inpJaewonCheck; }
			set { this._inpJaewonCheck = value; }
		}

		public MultiResultViewLayPaInfo() { }

		public MultiResultViewLayPaInfo(String suname, String suname2, String sex, String yearAge, String monthAge, String gubun, String admDictNm, String birth, String tel, String tel1, String telHp, String email, String zipCode1, String zipCode2, String address1, String address2, String inpJaewonCheck)
		{
			this._suname = suname;
			this._suname2 = suname2;
			this._sex = sex;
			this._yearAge = yearAge;
			this._monthAge = monthAge;
			this._gubun = gubun;
			this._admDictNm = admDictNm;
			this._birth = birth;
			this._tel = tel;
			this._tel1 = tel1;
			this._telHp = telHp;
			this._email = email;
			this._zipCode1 = zipCode1;
			this._zipCode2 = zipCode2;
			this._address1 = address1;
			this._address2 = address2;
			this._inpJaewonCheck = inpJaewonCheck;
		}

	}
}